bindings/python/src/connection.rs

```diff
@@ -110,39 +101,11 @@ impl Connection {
     /// con = glaredb.connect()
     /// con.sql('create table my_table (a int)').execute()
     /// ```
-    pub fn sql(&mut self, py: Python<'_>, query: &str) -> PyResult<PyLogicalPlan> {
-        let cloned_sess = self.session.clone();
+    pub fn sql(&mut self, py: Python<'_>, query: &str) -> PyResult<PyExecution> {
```

Comment:
does this need to take a mutable self `&mut self`? 

if we want to support:

> which has a few quirks in service of making it possible to reference python variables as tables in later SQL queries

wouldn't it make more sense to just clone `self.inner` on each call instead? It seems like a pretty cheap clone to make

Reply:
done